assets/prometheus-k8s/prometheus-rule.yaml

```diff
@@ -148,10 +148,10 @@ spec:
           )
         )
         * 100
-        > 1
+        > 10
```

Comment:
Updating expressions isn't in the scope of this PR, we should stick to only fixing the severity.